superset/assets/javascripts/explorev2/actions/exploreActions.js

```diff
@@ -85,3 +87,39 @@ export function changeFilterValue(filter, value) {
 export function setFieldValue(key, value) {
   return { type: SET_FIELD_VALUE, key, value };
 }
+
+export function updateViz(viz) {
```

Comment:
could change these names a bit for clarity? what do you think of `saveFormData`, `saveFormDataStarted`, `saveFormDataSucceeded`, `saveFormDataFailed`